docs/paper/admin/reference/configuration/commands-configuration.mdx

```diff
@@ -0,0 +1,55 @@
+---
+slug: /reference/commands-configuration
+---
+
+# Commands Configuration
+
+import React from 'react';
+import Config from '@site/src/components/Config';
+
+:::info
+
+The below YAML shows you the structure and default values for `commands.yml`.
+
+Click on a leaf node to view the description for that setting.
+
+:::
+
+<Config data={`
+command-block-overrides:
+  default: '[<command to override>]'
+  description: 'Which vanilla commands should be prioritized over those provided by bukkit or plugins.
+    Useful for compatibility with adventure maps built for vanilla command blocks.
+
+    Use the literal ''*'' to always use the vanilla version in command blocks.
+
+    By default, no commands are overridden.'
+ignore-vanilla-permissions:
+  default: 'false'
+  description: 'Whether to use vanilla permission levels when executing commands.'
+aliases:
+  '<alias name>':
+    default: '[<commands to run>]'
+    description: 'A List of Strings which are target commands.
+      Alternatively, a String, which is a single target command.
```

Comment:
I don't think this (`List`, `String`) should be uppercased like class names, people reading this won't care that it's a Java class